central/src/main/java/org/glowroot/central/repo/CentralRepoModule.java

```diff
@@ -93,19 +91,18 @@ public CentralRepoModule(ClusterManager clusterManager, Session session, File co
         traceAttributeNameDao = new TraceAttributeNameDao(session, configRepository, clusterManager,
                 targetMaxCentralUiUsers);
 
-        if (populateFromAdminDefault) {
-            File adminDefaultFile = new File(confDir, "admin-default.json");
-            if (adminDefaultFile.exists()) {
-                try {
-                    populateFromAdminDefault(adminDefaultFile, configRepository);
-                } catch (Exception e) {
-                    // drop the table so that after fixing admin-default.json re-import will occur
-                    session.updateSchemaWithRetry("drop table central_config");
-                    throw e;
-                }
-            }
-        }
+        File adminDefaultFile = new File(confDir, "admin-default.json");
 
+        if (adminDefaultFile.exists()) {
```

Comment:
this change will cause the central configuration to be overwritten on every startup

Reply:
Ok, this somehow went totally out of my scope. Sorry for the major delay here.

This would be needed in order for the helm chart to work and to be able to configure glowroot again when values have changed.
With using the helm chart we can easily deploy glowroot into k8s

Reply:
Would there be a possibility to somehow allow this? Maybe with an additional starting flag or something like this?

Reply:
ya, we'd need some way to know that the user wants to overwrite their config with `admin-default.json`.

what about just updating `admin.json` directly from the helm chart, instead of updating `admin-default.json`?

Reply:
that could work as well, I'll give it a shot and report back here :) Thanks for the suggestion!

Reply:
Ok, had some time to spare today and i tested a bit. However, this might be a stupid question, where exactly would the admin.json be located? I can't find it.

Reply:
oh! my bad, I got the standalone vs the central collector confused. `admin.json` is only for standalone. central collector stores the admin configuration in cassandra. 🤔

Reply:
Good, I'm not totally confused :D.
So the only way to configure a glowroot running cassandra is by using the admin-default. 
How about an external flag, which signals the user, that his config will always be overwritten by the admin-default. Something like a "helm-chart mode" :)

Reply:
@trask which way would you prefer?

Reply:
> How about an external flag, which signals the user, that his config will always be overwritten by the admin-default

this makes sense to me. in this mode `admin-default.json` would be the source of truth, and configuration will be overwritten with `admin-default.json` on each startup.